sdk/src/beta9/runner/common.py

```diff
@@ -93,6 +93,30 @@ def load_from_env(cls) -> "Config":
     config: Config = Config.load_from_env()
 
 
+class ConfigProxy:
```

Comment:
Not sure ConfigProxy is the right name for this

Reply:
I changed it to parent abstraction proxy. Bit more verbose but more accurate